src/lib.rs

```diff
@@ -93,8 +92,8 @@ impl<K, V, S> LruCache<K, V, S> where K: Eq + Hash, S: HashState {
     ///
     /// cache.insert(1, "a");
     /// cache.insert(2, "b");
-    /// assert_eq!(cache.get(&1), Some(&"a"));
-    /// assert_eq!(cache.get(&2), Some(&"b"));
+    /// assert_eq!(cache.get_mut(&1), Some(&mut "a"));
+    /// assert_eq!(cache.get_mut(&2), Some(&mut "b"));
```

Comment:
Unfortunately there's no `PartialEq` impl that'll work with just `&` when it's within an `Option`.